packages/design-system-web/src/components/geov-yasgui-map-circles/geov-yasgui-map-circles.tsx

```diff
@@ -226,29 +245,26 @@ export class GeovYasguiMapCircles {
   render() {
     return (
       <Host>
-        <ion-card style={{ display: 'none', zIndex: '1000' }}>
+        <ion-card class={`legend ${this.ledgendExpanded ? 'expanded' : 'collapsed'}`}>
           <ion-card-header>
-            <ion-card-title>Unable to render (some) results</ion-card-title>
+            <ion-button size="small" class="collapse-button" fill="clear" onClick={() => (this.ledgendExpanded = !this.ledgendExpanded)}>
+              <ion-icon icon={this.ledgendExpanded ? closeOutline : expandOutline} slot="icon-only"></ion-icon>
```

Comment:
expandOutline is the wrong icon in my opinion. it suggest fullscreen of the map. why not take an i-icon? because it's about information?

Reply:
agreed